.github/workflows/build.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
-          python-version: '3.8'
+          python-version: '3.9'
```

Comment:
This needs to be done on a branch off the 1.1.1 tag so 3.9 wheels can be built for that version.